include/MSound/MSoundBGM.hpp

```diff
@@ -35,6 +32,16 @@ class MSBgm : public JALListHioNode<MSBgm, u32> {
 	static u32 getSceneNo(u32);
 	static JAISound* getHandle(u8);
 	static JAISound* getJASTrack(JAISound*, u8);
+	//TODO: Found out where this inline is from.
+	static JAISound* someInline() {
+        if (smBgmInTrack[0] != 0) {
+    		return smBgmInTrack[0]->unk_14;
+    	} else {
+            return nullptr;
+        }
+    }
+public:
+	/* 0x14 */ JAISound* unk_14;
```

Comment:
rename to unk14 please for consistency
also clang-format the code and we can merge